tasks-by-users/samar4tech/style.css

```
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  padding: 0;
+
```

```suggestion
box-sizing: border-box;

```
 Add this css reset @samar4tech